inductiva/templates/heat_sink/openfoam/files/constant/initialConditions

```diff
@@ -0,0 +1,21 @@
+/*--------------------------------*- C++ -*----------------------------------*\
+  =========                 |
+  \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox
+   \\    /   O peration     | Website:  https://openfoam.org
+    \\  /    A nd           | Version:  8
+     \\/     M anipulation  |
+\*---------------------------------------------------------------------------*/
+FoamFile
+{
+    version         2;
+    format          ascii;
+    class           dictionary;
+    object          controlDict;
+}
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
+
+Tinitial        297;
```

Comment:
Because with a command I have only one file (`commands.json`) with configurable parameters. Setting this temperature in template files would require having at least three files.

Reply:
What I mean is that you could just change the temperature and the heaterPower here, because then this file will be used across all others. The command you are performing just changes things here, no?